CHANGELOG.md

```diff
@@ -86,4 +86,4 @@ This is the first CHANGELOG entry.
 ### Misc:
 * Update gems
 * Update bundler to 2.2
-* Update Ruby to 2.7
+* Update Ruby to 3.0
```

Comment:
```suggestion
* Update Ruby to 2.7
```
Lets revert this change. I'll add an entry to update the changelog after we get this merged.

Reply:
@kyleboe  Done

Reply:
@kyleboe  Done